packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-vector-settings.test.tsx

```diff
@@ -30,9 +33,8 @@ const defaultLabel = getDefaultFigureForType("label");
 // Mock the async function generateSpokenMathDetails
 jest.mock("./util", () => ({
     ...jest.requireActual("./util"),
-    generateSpokenMathDetails: (input) => {
-        return Promise.resolve(`Spoken math details for ${input}`);
-    },
+    joinLabelsAsSpokenMath: (input) =>
+        mockedJoinLabelsAsSpokenMathForTests(input),
```

Comment:
I worry that this mocking makes the tests harder to understand, and will become tech debt once we enable our wrapped SpeechRuleEngine to read locale data from local files.

Can you add a TODO comment linked to https://khanacademy.atlassian.net/browse/LEMS-2616 so we remember to un-mock this?